tasks/configure.yml

```diff
@@ -1,6 +1,28 @@
 ---
+
 - name: configure rabbitmq
   template:
     src: rabbitmq.config.j2
     dest: /etc/rabbitmq/rabbitmq.config
   when: rabbitmq_config is defined
+
+- name: rabbitmq full stop
+  include: node_fullstop.yml
+  when: rabbitmq_erlang_cookie is defined
+
+- name: set erlang cookie
+  template:
+    src: erlang.cookie.j2
+    dest: "{{ rabbitmq_erlang_cookie_file }}"
+    owner: rabbitmq
+    group: rabbitmq
+    mode: 0400
+  when: rabbitmq_erlang_cookie is defined
+
+- include: node_fullstart.yml
+
+- include: vhosts.yml
```

Comment:
This may fail when creating new hosts. Similar to #6, however, I have opened a separate issue for this: #7.

Reply:
now with ansible 2.4 there should be improvements on these rabbitmq functions. Also it should support TLS that was a requirements for me with this role, so until ansible 2.3 there were issues with TLS and certificates. Now those should be fixed. Not sure. I suspended working on this one.